lib/libva/DETAILS

```diff
@@ -1,11 +1,11 @@
           MODULE=libva
-         VERSION=2.1.0
-          SOURCE=$MODULE-$VERSION.tar.bz2
-      SOURCE_URL=https://github.com/01org/libva/releases/download/$VERSION/
-      SOURCE_VFY=sha256:f3fa953a11d3210c3a4ee79031abdbe0863d5ce13d9b3f93f315f1eec60a4b0f
+         VERSION=2.4.0
+          SOURCE=$MODULE-$VERSION.tar.gz
+ SOURCE_URL_FULL=https://github.com/intel/libva/archive/$VERSION.tar.gz
```

Comment:
Please use release instead of archive. eg https://github.com/intel/libva/releases/download/2.4.0/libva-2.4.0.tar.bz2